firstsection.css

```diff
@@ -0,0 +1,542 @@
+.first {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/final-Axion/pullRequest?prid=5455661)